### PR TITLE
Add support for the backtracer API

### PR DESCRIPTION
**Pull request by s1341**

### CODE REVIEW

**Comment by meme** [comment]
I think that this should be a gated feature with `backtrace`. This is because it adds `libc` as a runtime dependency, so it is not necessarily supported on all platforms we wish to support (see: Windows work).

**Comment by s1341** [comment]
Ok. Makes sense. Should we call the feature `backtrace` or `libc`. The latter would be useful if we add more `libc` dependent stuff....

**Comment by meme** [comment]
If you call it `libc` then we also have to gate this `Backtrace` with a platform flag (is it supported on macOS?). I am thinking we should call it `backtrace` and if `libc` comes up again we can change it. Let me know what you are thinking.

**Comment by s1341** [comment]
Ok. Will do.

**Comment by s1341** [comment]
I think this is sufficient to hide it. Ready to merge.

**Comment by meme** [comment]
It doesn't look like the `libc` dependency is gated on `backtrace`?

File: frida-gum/src/lib.rs [hunk 1]
```diff
@@ -70,6 +70,11 @@ pub use memory_range::*;
 mod range_details;
 pub use range_details::*;
 
+#[cfg_attr(backtrace, feature(backtrace))]
+mod backtracer;
+#[cfg_attr(backtrace, feature(backtrace))]
+pub use backtracer::*;
```
Comment (meme) [inline frida-gum/src/lib.rs:76]: `cfg_attr` is for documentation tags, not for enabling/disabling features. Instead use the following:
```
#[cfg(feature = "backtrace")]
#[cfg_attr(doc_cfg, doc(cfg(feature = "backtrace")))]
```

**Review by meme — CHANGES_REQUESTED** [review]
Please see comment and CI output

File: frida-gum/src/lib.rs (hunk 1 above)
Comment (s1341) [reply frida-gum/src/lib.rs:76]: Done.

**Comment by s1341** [comment]
Fixed. Sorry about that.

**Review by meme — APPROVED** [review]

**Pull request merged**